infra/charts/turing/README.md

```diff
@@ -1,42 +1,117 @@
 # turing
 
-![Version: 0.1.7](https://img.shields.io/badge/Version-0.1.7-informational?style=flat-square)
+---
```

Comment:
I suppose this file was generated from `infra/charts/turing/README.md.gotmpl`. How was it done?

Reply:
By using [norwoodj/helm-docs](https://github.com/norwoodj/helm-docs). I'm thinking about either adding a pre-commit hook to automate this or integrating this step into the CI.